src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -281,6 +314,7 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
       resourceLoader.bootConfig.resources.runtime[icuDataResourceName],
       'globalization'
     );
+    icuDataResource.response.then(_ => setProgress());
```

Comment:
One more nitpick - sorry about these :) I think calls like this can be simplified to:

```suggestion
    icuDataResource.response.then(setProgress);
```